helper/scss.js

```diff
@@ -5,7 +5,8 @@ module.exports = function(gulp, plugins, config, name, file) { // eslint-disable
         stylesDir   = theme.stylesDir ? theme.stylesDir : 'styles',
         disableMaps = plugins.util.env.disableMaps || false,
         production  = plugins.util.env.prod || false,
-        postcss     = [];
+        postcss     = [],
+        disableSuffix = theme.disableSuffix ? true : false;
```

Comment:
It can be reduced to `theme.disableSuffix || false` also keep the same aligning of `=` as above.